src/coreComponents/unitTests/fluidFlowTests/CMakeLists.txt

```diff
@@ -5,7 +5,6 @@
 set( gtest_geosx_tests
      testSinglePhaseBaseKernels.cpp
      testSinglePhaseFVMKernels.cpp
-     testSinglePhaseHybridFVMKernels.cpp
```

Comment:
Here I removed the unit test because with the new style (kernel class, etc), it becomes literally impossible to write a unit test for the assembly functions. Compositional solvers do have a unit test for their assembly functions, but they involve an xml hard-coded in the test, which is not really unitary. For single-phase flow, the derivatives are simple, so I thought it was just easier to remove the unit test.